opium/src/import.ml

```diff
@@ -16,6 +16,28 @@ module List = struct
       | Some _ as result -> result
       | None -> find_map ~f l)
   ;;
+
+  let replace_or_add_to_list ~f to_add l =
```

Comment:
Maybe this could be just `List.replace_or_add`? The `_to_list` suffix is now redundant.